index.js

```diff
@@ -1,47 +1,47 @@
 'use strict';
 
-let AWSXRay = require('aws-xray-sdk');
-let url_lib = require('url');
-let https = AWSXRay.captureHTTPs(require('https'));
-let AWS = AWSXRay.captureAWS(require('aws-sdk'));
-let s3 = new AWS.S3();
+const AWSXRay = require('aws-xray-sdk');
+const https = AWSXRay.captureHTTPs(require('https'));
+const AWS = AWSXRay.captureAWS(require('aws-sdk'));
```

Comment:
Codacy has a fix for the issue: [Replace `'aws-sdk'` with `"aws-sdk"`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
const AWS = AWSXRay.captureAWS(require("aws-sdk"));
```